examples/riscv_mini/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest
```

Comment:
do conftest.py's compose hierarchically? i.e. how does this file interact with top-level conftest.py?

Reply:
Yes, a local conftest will run only for the tests in a sub-directory: https://docs.pytest.org/en/6.2.x/writing_plugins.html#conftest-py-local-per-directory-plugins (so rather than thinking about it interacting with the top-level conftest, it may be more useful to think of it as "scoping" configuration for sub-directories).